<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinanceCalculator</class>
 <widget class="QWidget" name="FinanceCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="summ_6">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Output sum</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_period_credit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>105</y>
     <width>521</width>
     <height>50</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Period (in months)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_a">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>325</y>
     <width>156</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Аннуитетный кредит</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_d">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>265</y>
     <width>217</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференцированный кредит</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_overpayment_credit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>420</y>
     <width>231</width>
     <height>33</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_sum_credit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>370</y>
     <width>231</width>
     <height>33</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate_credit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate credit</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="month_table">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>25</y>
     <width>611</width>
     <height>331</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>2</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>Month</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Payout</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="input_rate_credit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>521</width>
     <height>50</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string> Rate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_sum_credit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>521</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string> Sum</string>
   </property>
  </widget>
  <widget class="QLabel" name="summ_10">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
  </widget>
  <widget class="QLabel" name="summ_11">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>430</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>amount</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
